Necrosis/Spells.lua

```diff
@@ -258,96 +258,11 @@ function Necrosis:SpellLocalize(tooltip)
-			[25] = {11719,1714},											-- Curse of Tongues || Malédiction Langage
-			[26] = {11722,11721,1490},										-- Curse of the Elements || Malédiction Eléments
-			[27] = {},														-- NOPE NOT IN Classic  Metamorphosis || Metamorphose
-			[28] = {18881,18880,18879,18265},								-- Siphon Life || Syphon de vie
-			[29] = {17928,5484},											-- Howl of Terror || Hurlement de terreur
-			[30] = {18540},													-- Ritual of Doom || Rituel funeste
-			[31] = {11735,11734,11733,1086,706},							-- Demon Armor || Armure démoniaque
-			[32] = {5697},													-- Unending Breath || Respiration interminable
-			[33] = {11743,2970,132},										-- Detect Invisibility || Détection de l'invisibilité
-			[34] = {126},													-- Eye of Kilrogg
-			[35] = {11726,11725,1098},										-- Enslave Demon
-			[36] = {696,687},												-- Demon Skin || Peau de démon 
-			[37] = {698},													-- Ritual of Summoning || Rituel d'invocation
-			[38] = {19028},													-- Soul Link || Lien spirituel
-			[39] = {},														-- NOPE NOT IN Classic  Demon Charge || Charge démoniaque
-			[40] = {18223},													-- Curse of Exhaustion || Malédiction de fatigue
-			[41] = {11689,11688,11687,1456,1455,1454},						-- Life Tap || Connexion
-			[42] = {},														-- NOPE NOT IN Classic  Haunt || Hanter
-			[43] = {28610,11740,11739,6229},								-- Shadow Ward || Gardien de l'ombre
-			[44] = {18788},													-- Sacrifice || Sacrifice démoniaque 
-			[45] = {25307,11661,11660,11659,7641,1106,1088,705,695,686},	-- Shadow Bolt
-			[46] = {},														-- NOPE NOT IN Classic  Unstable Affliction || Affliction instable
-			[47] = {},														-- NOPE NOT IN Classic  Fel Armor || Gangrarmure
-			[48] = {},														-- NOPE NOT IN Classic  Seed of Corruption || Graine de Corruption
-			[49] = {},														-- NOPE NOT IN Classic SoulShatter || Brise âme
-			[50] = {},														-- NOPE NOT IN Classic Ritual of Souls || Rituel des âmes
-			[51] = {20757,20756,20755,20752,693},							-- Create Soulstone || Création pierre d'âme
-			[52] = {11730,11729,5699,6202,6201},							-- Create Healthstone || Création pierre de soin
-			[53] = {17728,17727,2362},										-- Create Spellstone || Création pierre de sort
-			[54] = {17953,17952,17951,6366},								-- Create Firestone || Création pierre de feu
-			[55] = {18938,18937,18220},										-- Dark Pact || Pacte noir
-			[56] = {},														-- NOPE NOT IN Classic  Shadow Cleave || Enchainement d'ombre
-			[57] = {},														-- NOPE NOT IN Classic  Immolation Aura || Aura d'immolation
-			[58] = {},														-- NOPE NOT IN Classic  Challenging Howl || Hurlement de défi
-			[59] = {}														-- NOPE NOT IN Classic   Demonic Empowerment || Renforcement démoniaque
-		}
+function Necrosis:ManaCostLocalize(spellIndex)
+	if GetLocale() == "ruRU" then
+		GameTooltip:AddLine(self.Translation.Misc.Mana..": "..self.Spell[spellIndex].Mana)
```

Comment:
Unless we are updating mana values somewhere we only set to 50 in the object creation.